intel-retail/automated-self-checkout
# fix: have non helm version use local sample video when available

## PR Checklist



## What are you changing?

Mount local sample-media to the docker container. If the target video isn't found in the volume the new container launch script will download the video directly.

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Make run-demo
- See video running as usual
- Delete sample videos
- RENDER_MODE=1 docker compose -f src/docker-compose.yml up -d
- See video output the same as run-demo

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
